README.md

```diff
@@ -7,43 +7,42 @@
 
 ## Table of contents
```

Comment:
Why remove the dot at the end of each item?

Reply:
Because it is link now. I think the links end without dot. Right?